doc/HACKING.md

```diff
@@ -168,12 +171,16 @@ pip3 install --user \
          -r contrib/pyln-testing/requirements.txt
 ```
 
-Re-run `configure` for the python dependencies
+Re-run `configure` for the python dependencies and build using `make`.
 
 ```
 ./configure --enable-developer
+make -j
```

Comment:
??
```suggestion
make -j$(nproc)
```

Reply:
If you insist I can change it, but `man make` says that it's the same without argument. This has been for this for a while now:

```
If the -j option is given without an argument, make will not limit the number of jobs that can run simultaneously.
```

Reply:
You can grind your machine into the ground with make -j, no?

Reply:
Looks like I didn't even read the manage quote correctly :D. I always assumed it starts as many jobs as #CPUs without arg. Will check tomorrow...

Reply:
@niftynei sorry for my stupid comment.  :D fixed.